src/tasks/installers/operator.ts

```diff
@@ -32,6 +32,21 @@ export class OperatorTasks {
   dwRoutingCRD = 'devworkspaceroutings.controller.devfile.io'
   legacyClusterResourcesName = 'che-operator'
 
+  // Delete this method and use default v1 CRDs when Openshift 3.x support dropped
+  private async getBackupRestoreCrdFilesNames(kube: KubeHelper): Promise<[string, string]> {
```

Comment:
Could you move function close to  `getCRDPath(ctx: any, flags: any)`
Having private function in the beginning is not a good approach.

Reply:
ok